pkg/pods/pod.go

```diff
@@ -76,6 +77,9 @@ type Pod struct {
 
 	// Pod will not start if file is not present
 	RequireFile string
+
+	// subsystemer is a tool for this pod to find its cgroup subsystem controller and metadata. Optionally nil, overridden in test
```

Comment:
teeny-nit: is this comment a little misleading? I'm under the impression that `subsystemer` cannot technically be `nil`, only zero-valued